tests/application/admin/test_views.py

```diff
@@ -392,3 +394,21 @@ def test_admin_user_can_see_all_data_sources(self, test_app_client, logged_in_ad
 
         assert "Police statistics 2019" in page.text
         assert "2011 Census of England and Wales" in page.text
+
+    def test_admin_user_search_for_data_sources(self, test_app_client, logged_in_admin_user):
+
+        DataSourceFactory.create(title="Police statistics 2019")
+        DataSourceFactory.create(title="2011 Census of England and Wales")
+
+        response = test_app_client.get("/admin/data-sources?q=police")
+        page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
+
+        assert response.status_code == 200
+        assert "Data sources" == page.find("h1").text
+        assert "police - Search data sources" == page.find("title").text
+
+        assert "Police statistics 2019" in page.text
+        assert "2011 Census of England and Wales" not in page.text
```

Comment:
These `in page.text` assertions always feel overly broad to me ... could we maybe limit it at least to the main content of the page? `in page.find('main').text`?